src/pages/api/utils/sendMailToMyContacts.ts

```diff
@@ -0,0 +1,65 @@
+import { IExecWeb3mail, getWeb3Provider as getMailProvider, Contact } from '@iexec/web3mail';
+import {
+  IExecDataProtector,
+  ProtectedData,
+  getWeb3Provider as getProtectorProvider,
+} from '@iexec/dataprotector';
+
+//TODO: Add filters ?
+export const sendMailToMyContacts = async (
```

Comment:
may be we can think of a different name as now it's not only my contacts, it's potentially a list of addresses